src/chains/const/index.ts

```diff
@@ -18,6 +19,8 @@ export const ALL_SUPPORTED_CHAINS_MAP: Record<SupportedChainId, ChainInfo> = {
   [SupportedChainId.ARBITRUM_ONE]: arbitrumOne,
   [SupportedChainId.BASE]: base,
   [SupportedChainId.SEPOLIA]: sepolia,
+  [SupportedChainId.AVALANCHE]: avalanche,
+  [SupportedChainId.POLYGON]: polygon,
```

Comment:
Nitpick: Leave sepolia last. 
The reasoning being:
- First L1s (mainnet, gnosis)
- Then L2s
- Last testnets